test/functional/wallet_createwallet.py

```diff
@@ -96,5 +96,22 @@ def run_test(self):
         assert_raises_rpc_error(-4, "Error: This wallet has no available keys", w5.getnewaddress)
         assert_raises_rpc_error(-4, "Error: This wallet has no available keys", w5.getrawchangeaddress)
 
+        self.log.info('New blank and encrypted wallets can be created')
+        self.nodes[0].createwallet(wallet_name='wblank', disable_private_keys=False, blank=True, passphrase='thisisapassphrase')
+        wblank = node.get_wallet_rpc('wblank')
+        wblank.walletpassphrase('thisisapassphrase', 10)
+
+        self.log.info('Test creating a new encrypted wallet.')
+        # Born encrypted wallet is created (has keys)
+        self.nodes[0].createwallet('w6', False, False, 'thisisapassphrase')
+        w6 = node.get_wallet_rpc('w6')
+        w6.walletpassphrase('thisisapassphrase', 10)
```

Comment:
again, I suggest testing that the wallet is locked immediately after creation (this test will currently fail).

Reply:
Done